### PR TITLE
Aymissouri create post

### PR DESCRIPTION
**Pull request by ayMissouri**
🙏

### CODE REVIEW

File: src/components/createPostModal/index.js [hunk 1]
```diff
@@ -2,32 +2,67 @@ import { useState } from "react"
...
+
+    const name = `${props.user.firstName} ${props.user.lastName}`
+    const initials = name.match(/(\b\S)?/g).join("").match(/(^\S|\S$)?/g).join("").toUpperCase()
+
     return (
         <>
             <section className="create-post-user-details">
-                <div className="profile-icon"><p>AJ</p></div>
-                <div className="post-user-name"><p>Alex J</p></div>
+                <div className="profile-icon"><p>{initials}</p></div>
```
Comment (chris2733) [inline src/components/createPostModal/index.js]: Missing $ from the start

File: src/components/createPostModal/style.css [hunk 2]
```diff
@@ -12,3 +12,11 @@ textarea {
     border-radius: 8px;
     padding: 16px;
 }
+.error
+{
```
Comment (chris2733) [inline src/components/createPostModal/style.css]: Bit picky but get these brackets up at the end of the classname, for consistency across the project

File: src/pages/dashboard/index.js [hunk 3]
```diff
@@ -7,9 +7,41 @@ import TextInput from "../../components/form/textInput";
...
+                "bio": "",
+                "githubUrl": ""
+            }
+        }
+    })
+
+	useEffect(()=>{
+		const getUserInfo = async () => {
+			const res = await get(`users/${userId}`)
+			setUser(res.data.user)
```
Comment (chris2733) [inline src/pages/dashboard/index.js]: This file needs to be formatted

**Review by chris2733 — CHANGES_REQUESTED** [review]

File: src/components/createPostModal/index.js [hunk 4]
```diff
@@ -2,32 +2,67 @@ import { useState } from "react"
...
+
+    const name = `${props.user.firstName} ${props.user.lastName}`
+    const initials = name.match(/(\b\S)?/g).join("").match(/(^\S|\S$)?/g).join("").toUpperCase()
+
     return (
         <>
             <section className="create-post-user-details">
-                <div className="profile-icon"><p>AJ</p></div>
-                <div className="post-user-name"><p>Alex J</p></div>
+                <div className="profile-icon"><p>{`${initials}`}</p></div>
```
Comment (chris2733) [inline src/components/createPostModal/index.js]: This can just be ${initials}, same with the lower line

**Review by chris2733 — CHANGES_REQUESTED** [review]

**Review by chris2733 — APPROVED** [review]

**Review by MaybeFreak — APPROVED** [review]
I will trust you :)

**Pull request merged**